plutus-extra/plutus-extra.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               plutus-extra
-version:            1.0
+version:            1.0.1.0
```

Comment:
This version number is invalid according to the [style guide](https://github.com/mlabs-haskell/styleguide#versioning).

Reply:
Changed it to `1.1`.